playbooks/roles/logstash/tasks/logstash_post_install.yml

```diff
@@ -13,28 +13,24 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-- name: Deploy ElasticSearch configuration files
-  template:
-    src: "{{ item }}"
-    dest: "/etc/logstash/conf.d/{{ item }}"
+- name: Prevent logstash-web from starting on boot
+  copy:
+    dest: "/etc/init/logstash-web.override"
+    content: "manual\n"
```

Comment:
Not sure about that one, but we don't want logstash-web to start so this just moves it out of the way.